scholarly_articles/tasks.py

```diff
@@ -215,3 +221,161 @@ def load_crossref(from_update_date=2012, until_update_date=2012):
 
     except Exception as e:
         logger.info(f"Unexpected error: {e}")
+
+
+@celery_app.task(name=_("Load sucupira data"))
+def load_sucupira(file_path, user_id):
+    """
+    Load the data from sucupira file.
+
+    Sync or Async function
+
+    Param file_path: String with the path of the .csv file.
+    Param user: The user id passed by kwargs on tasks.kwargs
+
+    Fields used on this load: 
+    
+        doi = DS_DOI
+        id_int_production
+        title = NM_PRODUCAO
+        volume = NR_VOLUME
+        number = NR_SERIE
+        year = AN_BASE_PRODUCAO
+        open_access_status = NA
+        use_license = CC-By
+        license = https://opendefinition.org/licenses/cc-by/
+        apc = NA
+        contributors = DICT_AUTORES
+        journal = DS_ISSN (refe to journal)
+        source = SUCUPIRA
+
+    """
+
+    def get_institution(institution_acron):
+        """
+        Get the institution from models.Institution.
+
+        Example of the acronym of institution:
+
+            UFPA - ABAETETUBA
+            UFPB/AREIA
+            UNESP-ARAR
+            UTFPR-MD
+            CCD/SES
+            CEFET/MG
+        """
+
+        # makes a treatment to obtain the institution
+        institution_acron = institution_acron.split("/")[0]
+        institution_acron = institution_acron.split(" / ")[0]
+        institution_acron = institution_acron.split("-")[0]
```

Comment:
@gitnnolabs e se o hífen é parte do nome? Talvez neste momento não seja ideal tomar nenhuma decisão de padronização de nome.

Reply:
Ai nesse caso se não for encontrado a instituição na nossa base de dados não cadastramos a instituição? 

**Fiz esse tratamento para que a instituição seja encontrada na nossa base de dados.**

Reply:
Alterei para não fazer essa tratamento, iremos encontrar menos instituições mas fica mais consistente.